cpp/tests/test_abm_world.cpp

```diff
@@ -89,20 +91,19 @@ TEST(TestWorld, findLocation)
     auto home_id   = world.add_location(mio::abm::LocationType::Home);
     auto school_id = world.add_location(mio::abm::LocationType::School);
     auto work_id   = world.add_location(mio::abm::LocationType::Work);
-    auto person  = mio::abm::Person(home_id, mio::abm::InfectionState::Recovered_Carrier, mio::abm::AgeGroup::Age60to79,
-                                    world.get_global_infection_parameters());
-    auto& home   = world.get_individualized_location(home_id);
-    auto& school = world.get_individualized_location(school_id);
-    auto& work   = world.get_individualized_location(work_id);
+    auto& home     = world.get_individualized_location(home_id);
+    auto& school   = world.get_individualized_location(school_id);
+    auto& work     = world.get_individualized_location(work_id);
+    auto person    = create_person_simple(home);
     person.set_assigned_location(home);
     person.set_assigned_location(school);
     person.set_assigned_location({0, mio::abm::LocationType::Work});
 
-    ASSERT_EQ(world.find_location(mio::abm::LocationType::Work, person), &work);
-    ASSERT_EQ(world.find_location(mio::abm::LocationType::School, person), &school);
-    ASSERT_EQ(world.find_location(mio::abm::LocationType::Home, person), &home);
+    ASSERT_EQ(&world.find_location(mio::abm::LocationType::Work, person), &work);
```

Comment:
since you added `Location::operator==`, you could leave out the `&` address operator completely.